src/cmd/top/summary_pane.c

```diff
@@ -358,13 +358,16 @@ static int get_instance_attr_int (flux_t *h, const char *attr)
 }
 
 
-static int resource_count (json_t *o,
+static int resource_count (struct summary_pane *sum,
+                           json_t *o,
                            const char *name,
                            int *nnodes,
                            int *ncores,
                            int *ngpus)
```

Comment:
Minor, but since this function is counting resources returned from `sched.resource-status` maybe the function declaration would be more clear with the response payload first (as before), then add a new `json_t *constraint` argument rather than having the caller pass in all of `struct summary_pane` when it's only used to find the constraint?